### PR TITLE
Changed size and Removed Extra SVG

### PR DESCRIPTION
**Pull request by Harshit-Dandriyal**
Signed-off-by: Harshit Dandriyal <56961488+Harshit-Dandriyal@users.noreply.github.com>**Description**

This PR fixes #
Changed size and Removed Extra SVG
**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by Abhishek-kumar09 — APPROVED** [review]
LGTM

**Comment by Abhishek-kumar09** [comment]
Oh @Harshit-Dandriyal You forgot to add the utf encoding string

**Comment by Abhishek-kumar09** [comment]
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg>

ref: https://js.cytoscape.org/#style/background-image

**Review by Abhishek-kumar09 — APPROVED** [review]
LGTM

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
The naming convention of svgs are still screwed. Please do the required change in all the places and subsequently make changes in meshery-extensions.
Screenshot 2022-10-13 at 1 29 48 PM

**Comment by leecalcote** [comment]
Bump. Poke. Smack.

**Comment by Harshit-Dandriyal** [comment]
This PR is on little hold as we are on the way to remove the other SVG's here too.

**Comment by leecalcote** [comment]
> This PR is on little hold as we are on the way to remove the other SVG's here too.

Thanks for confirming. Please update the labels accordingly so that this fact is obvious.

File: install/Makefile.core.mk [hunk 1]
```diff
@@ -19,7 +19,7 @@ GIT_VERSION	= $(shell git describe --tags `git rev-list --tags --max-count=1`)
 GIT_COMMITSHA = $(shell git rev-list -1 HEAD)
 GIT_STRIPPED_VERSION=$(shell git describe --tags `git rev-list --tags --max-count=1` | cut -c 2-)
 
-GOVERSION = 1.19.1
+GOVERSION =
```
Comment (Abhishek-kumar09) [inline install/Makefile.core.mk:22]: 😖

**Comment by Abhishek-kumar09** [comment]
@Harshit-Dandriyal is it still `do-not-merge`

**Review by Abhishek-kumar09 — APPROVED** [review]
Looks Good to Me . Thanks @Harshit-Dandriyal Great Job here 🥳

**Review by Abhishek-kumar09 — APPROVED** [review]

**Pull request merged**